spacewalk/certs-tools/mgr_ssl_cert_setup.py

```diff
@@ -319,6 +319,8 @@ def getCertData(cert):
         elif line.startswith("    "):
             if nextval == "subjectKeyIdentifier":
                 data["subjectKeyIdentifier"] = line.strip().upper()
+            elif nextval == "authorityKeyIdentifier" and line.startswith("    keyid:"):
```

Comment:
What do you think of this approach? Maybe we just need to keep both if branches.

Reply:
We need 2 variants of parsing, but not this way. The second elif will always overwrite the keyid with the values after it and also match "   DirName:...." or "   Serial:....".
We need to check that the line start with spaces followed by 2 hex digits and a colon.

Reply:
I think I have a solution:

```
            elif nextval == "authorityKeyIdentifier" and re.match("^\s+[0-9A-Fa-f]{2}:.+$", line):
                data["authorityKeyIdentifier"] = line.strip().upper()
```